include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -558,15 +585,12 @@ struct SPhysicalDeviceLimits
     /* TexelBufferAlignmentFeaturesEXT *//* VK_EXT_texel_buffer_alignment */
     bool texelBufferAlignment = false;
 
-    // [TODO] implement the enable by default and expose behaviour for ones below once API changes
-
     /* ShaderSMBuiltinsFeaturesNV *//* VK_NV_shader_sm_builtins */
     bool shaderSMBuiltins = false;
 
     bool shaderSubgroupPartitioned = false; /* VK_NV_shader_subgroup_partitioned */
     bool gcnShader = false; /* VK_AMD_gcn_shader */
     bool gpuShaderHalfFloat = false; /* VK_AMD_gpu_shader_half_float */
-    bool gpuShaderInt16 = false; /* VK_AMD_gpu_shader_int16 */
```

Comment:
what happened to this?

Reply:
Deprecated by [VK_KHR_shader_float16_int8](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_KHR_shader_float16_int8.html) extension

Which in turn was promoted to [Vulkan 1.2](https://registry.khronos.org/vulkan/specs/1.3-extensions/html/vkspec.html#versions-1.2-promotions)

Reply:
Functionality in this extension was included in [VK_KHR_shader_float16_int8](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_KHR_shader_float16_int8.html) extension, when [VkPhysicalDeviceFeatures](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkPhysicalDeviceFeatures.html)::shaderInt16 and [VkPhysicalDeviceShaderFloat16Int8FeaturesKHR](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkPhysicalDeviceShaderFloat16Int8FeaturesKHR.html)::shaderFloat16 are enabled.

Reply:
ok